frontend/micro-ui/web/micro-ui-internals/packages/modules/cases/src/pages/employee/JoinCaseHome.js

```diff
@@ -5,17 +5,18 @@ import {
   CheckSvg,
   CloseSvg,
   Dropdown,
+  FormComposerV2,
   LabelFieldPair,
   RadioButtons,
   TextInput,
 } from "@egovernments/digit-ui-react-components";
 import { useHistory } from "react-router-dom";
-import React, { useEffect, useState } from "react";
-import { Link } from "react-router-dom";
+import React, { useEffect, useRef, useState } from "react";
 import { InfoCard } from "@egovernments/digit-ui-components";
 import { DRISTIService } from "../../../../dristi/src/services";
 import { RightArrow } from "../../../../dristi/src/icons/svgIndex";
 import { CASEService } from "../../hooks/services";
+import { isEqual } from "lodash";
```

Comment:
replace with `import isEqual from "lodash/isEqual";`

Reply:
code isn't even compiling

Reply:
I made the changes.